<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LuaFrontend [v0.75 | BETA]</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="mainGroup" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gameGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Games:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="gameWidget">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="scriptGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>6</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Scripts:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="scriptWidget">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="autoScrollMargin">
             <number>16</number>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>3</number>
            </property>
            <attribute name="headerCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="headerMinimumSectionSize">
             <number>35</number>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">3</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mainMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEngine">
    <property name="title">
     <string>Engine</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionConsole"/>
    <addaction name="actionAutoReload"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionTK"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionDark"/>
   </widget>
   <addaction name="menuEngine"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionConsole">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Console...</string>
   </property>
  </action>
  <action name="actionAutoReload">
   <property name="text">
    <string>Enable Auto-Reload</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start Engine</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reload Engine</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop Engine</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Switch to Dark Mode</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About LuaFrontend...</string>
   </property>
  </action>
  <action name="actionTK">
   <property name="text">
    <string>About TopazTK...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
